VideoWeb/VideoWeb/ClientApp/src/app/app.component.ts

```diff
@@ -62,20 +59,9 @@ export class AppComponent implements OnInit, OnDestroy {
         pageTracker.trackPreviousPage(router);
     }
 
-    private initAuthentication() {
-        const clientSettings = this.configService.getClientSettings();
-        const config = {
-            tenant: clientSettings.tenant_id,
-            clientId: clientSettings.client_id,
-            postLogoutRedirectUri: clientSettings.post_logout_redirect_uri,
-            redirectUri: clientSettings.redirect_uri,
-            cacheLocation: 'sessionStorage'
-        };
-        this.adalService.init(config);
-    }
-
-    ngOnInit() {
-        this.checkAuth().then(() => {
+    async ngOnInit() {
```

Comment:
Ng does not respect async for `ngOnInit`.

Reply:
fixed